index.js

```diff
@@ -169,7 +181,7 @@ minimum value for how long a non-cached analyses will take
     try {
       analyses = await simpleRequester.do({ url, accessToken: this.accessToken, json: true })
     } catch (e) {
-      if (e.status !== 401) {
+      if (e.hasOwnProperty('statusCode') && e.statusCode !== 401) {
```

Comment:
`simpleRequester` throws error wrapped with `http-errors` library. Correct value for status code is `status`